pkg/runtime/pubsub/subscriptions.go

```diff
@@ -199,7 +199,8 @@ func GetSubscriptionsGRPC(channel runtimev1pb.AppCallbackClient, log logger.Logg
 				s, ok := status.FromError(rErr)
 				if ok && s != nil {
 					if s.Code() == codes.Unimplemented {
-						return rResp, backoff.Permanent(rErr)
+						log.Infof("pubsub subscriptions: gRPC app does not implement ListTopicSubscriptions")
+						return rResp, nil
```

Comment:
What is rResp in this case? `nil`? Should we just return `nil` (or another zero value) here?

Reply:
It is not nil, but I am changing this and the existing code to guarantee that and not depend on this behavior from the downstream function.